### DISCUSSION TITLE
FYI: Seems that ChatGPT API usage is not free (anymore)

### DISCUSSION THREAD

**Discussion by TommiPrami** [AliDehbansiahkarbon/ChatGPTWizard#38] [Show and tell]
I get 

"You exceeded your current quota, please check your plan and billing details"

message now , seems that if you need to use API, then have to pay something to use it.

**Comment by AliDehbansiahkarbon** [comment]
Actually, this is not a bug or issue with the plug-in.
The Plug-in is just a client to consume their API and is **free forever** but you need to upgrade your plan on [OpenAI's website](https://openai.com/pricing), with a reasonable price, that's not free forever.

**Reply by TommiPrami** [reply]
Sure this is not a big, That is why I put FYI, in the tittle. Maybe add this info to readme, if you feel like it .

Just noticed this with other app first, then tested this plugin and result is the same, my trial has edänded and would need to pay, that's all, but bet many do not know this.

**Reply by AliSawari** [reply]
> Sure this is not a big, That is why I put FYI, in the tittle. Maybe add this info to readme, if you feel like it .
> 
> Just noticed this with other app first, then tested this plugin and result is the same, my trial has edänded and would need to pay, that's all, but bet many do not know this.


The README clearly says **You need an API key to use this plugin**. I'm not sure how much it can be stressed enough.

**Reply by ChristianDenk772** [reply]
> > Sure this is not a big, That is why I put FYI, in the tittle. Maybe add this info to readme, if you feel like it .
> > Just noticed this with other app first, then tested this plugin and result is the same, my trial has edänded and would need to pay, that's all, but bet many do not know this.
> 
> The README clearly says **You need an API key to use this plugin**. I'm not sure how much it can be stressed enough.

In his comment he talked about the fact that only the trial key is free, but then it has to be paid. He never talked about needing an API key at all.
You should read the comments CAREFULLY before treating users in such a condescending manner  "I'm not sure how much it can be stressed enough".

**Comment by TommiPrami** [comment]
API key is totally different thing than it being free only for triaö period.

Things like these tends to make people bark at the wrong tree, when it
fails.

At least I didn't know that using API and API-key was not free. It took
some time to investigate.

I loose nothing if the info is not added to site/plugin/help etc, I already
know.

So take it or leave it.

-tee-

**Comment by AliDehbansiahkarbon** [comment]
This subject was mentioned before.
![image](https://github.com/AliDehbansiahkarbon/ChatGPTWizard/assets/5601608/8e6aa910-be8a-422d-8326-ce80ccafc2ea)


But it has been added in the Remark section too.

![image](https://github.com/AliDehbansiahkarbon/ChatGPTWizard/assets/5601608/6bb9b5d0-f3de-4a61-9402-8349602370d4)
